recipe/conda_build_config.yaml

```diff
@@ -11,13 +6,17 @@ MACOSX_SDK_VERSION:            # [osx and qt_version == 6]
 
 # As of 2024/03 -- OSX basically never had qt builds. If somebody asks, we can build them
 # As of 2024/03 -- qt6 works better than qt5 on aarch64. So drop qt5.
-# As of 2024/03 -- No windows users have expressed concerns about the qt libraries. don't build None
 # As of 2024/05 -- We had trouble gettin the aarch64 builds to work well with Qt6 and import cv2 on the CIs.
 #                  We can re-consider adding Qt6 support for Aarch upon request.
 qt_version:
-  - none       # [not win]
+  - none
```

Comment:
I want the headless build to make building other packages faster and to avoid pulling in Qt and all its dependencies accidentally in other packages.

Reply:
never mind too hard.